modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/.npmbundlerrc

```diff
@@ -1,5 +1,5 @@
 {
 #if (${buildType} != "gradle")
-    "output": "target/classes/META-INF/resources",
-#end    "preset": "liferay-npm-bundler-preset-angular"
+    "output": "target/classes/META-INF/resources"
+#end
```

Comment:
So we no longer need `"liferay-npm-bundler-preset-angular"`?

Reply:
Yep. its already included in bundler 2.0.0 

https://github.com/liferay/liferay-npm-build-tools/wiki/Migrating-bundler-from-1.x-to-2.x#3-now-remove-all-liferay-npm-bundler-preset--dependencies-as-bundler-2x-already-comes-with-those-inside